_scripts/make_static_libs.sh

```diff
@@ -61,6 +61,28 @@ ${CMAKE} \
 ${MAKE}
 ${MAKE} install
 
+# freetype
+GITCLONE https://git.savannah.nongnu.org/git/freetype/freetype2.git freetype2 VER-2-12-1
+# Freetype does not allow building inside source dir
+${CMAKE} -E make_directory build
+
+${CMAKE} -E chdir build \
+${CMAKE} \
+-DCMAKE_C_FLAGS="${MYCFLAGS}" \
+-DZLIB_INCLUDE_DIR=${INCLUDEDIR} \
+-DZLIB_LIBRARY_RELEASE=${LIBDIR}/libz.a \
+-DPNG_PNG_INCLUDE_DIR=${INCLUDEDIR} \
+-DPNG_LIBRARY_RELEASE=${LIBDIR}/libpng.a \
+-DFT_DISABLE_BROTLI=TRUE \
+-DCMAKE_INSTALL_PREFIX=${WORKDIR} \
+..
+
+${CMAKE} -E chdir build \
+${MAKE}
+
+${CMAKE} -E chdir build \
```

Comment:
Is this not redundant after the first one?

Reply:
No, it runs the subsequence command in context of the command. It's a helper from cmake to avoid changing dirs in the shell